#NAV_MSGS
Header header
float32 WGS84_Lat
float32 WGS84_Lon
float32 WGS84_Alt
float32 NED_N
float32 NED_E
float32 NED_D
float32 STD_DEV_LAT
float32 STD_DEV_LON
float32 STD_DEV_ALT
uint32	POSTYPE
